tasks/run.js

```diff
@@ -155,28 +155,9 @@ function ensureRouterInstall (router) {
 		.catch(function () { throw new Error(`You need to install the ${router} first!  See docs here: https://github.com/Financial-Times/${router}`); });
 }
 
-// Remind developers that if they want to use a local version of n-ui,
-// they need to `export NEXT_APP_SHELL=local`.
-function devNui () {
-	if (
-		(!process.env.NODE_ENV || process.env.NODE_ENV !== 'production') // Not production
-		&& !process.env.CIRCLE_BRANCH // Not CircleCI
-		&& (!process.env.NEXT_APP_SHELL || process.env.NEXT_APP_SHELL !== 'local') // NEXT_APP_SHELL is not set to local
-	) {
-		// Check if the app is using n-ui
-		shell('grep -s -Fim 1 n-ui bower.json')
```

Comment:
Linting is complaining about this being imported but not used.

Reply:
Fixed.